invenio.cfg

```diff
@@ -525,4 +526,15 @@ _APP_RDM_STATS_AGGREGATIONS["record-view-agg"]["params"]["index_interval"] = "ye
 
 APP_RDM_RECORD_LANDING_PAGE_TEMPLATE = "cds_rdm/records/detail.html"
 
-RDM_DETAIL_SIDE_BAR_MANAGE_ATTRIBUTES_EXTENSION_TEMPLATE = "cds_rdm/records/manage_menu.html"
\ No newline at end of file
+RDM_DETAIL_SIDE_BAR_MANAGE_ATTRIBUTES_EXTENSION_TEMPLATE = "cds_rdm/records/manage_menu.html"
+
+APP_RDM_RECORD_EXPORTERS = {
+    **RECORD_EXPORTERS,
+    "bibtex-allversions": {
+        "name": _("BibTeX (all versions)"),
```

Comment:
Nice, however I don't think it is clear enough.

```suggestion
        "name": _("BibTeX (DOI for all versions)"),
```

On top of that, I would also override the default BibTeX item and rename it.

Something like this?
![Screenshot 2025-07-16 at 11 28 39](https://github.com/user-attachments/assets/87e06518-622b-4218-b41e-a8931e59df1f)